### DISCUSSION TITLE
Favicon request in https

### DISCUSSION THREAD

**Discussion by jdanker** [adityatelange/hugo-PaperMod#1028] [General]
Hi, currently the favicons are being requested using http, and thus being blocked by most browsers. This causes the following error: 
![image](https://user-images.githubusercontent.com/31894119/187570918-a941a116-fbf9-4cdb-bdad-bad9781893a2.png)

Currently I have the favicons placed in the /static folder. 

Is there any way to have the favicons rendered with https in the head? Taking a brief look over the code I am not entirely sure why it wouldn't be. 

I'm new to hugo so any insight is appreciated, thanks!

**Comment by adityatelange** [comment]
Can u share code?

Its mainly baseurl, set baseurl as https://yoursite.com/

**Reply by jdanker** [reply]
Thanks for the quick reply! sure, my repo is [here](https://github.com/jdanker/jdanker-core) for my site. And my baseurl is already configured like that. (You can see the error in dev tools when loading my site: jdanker.com )

**Reply by adityatelange** [reply]
I do see https with favicon.ico

view-source:https://www.jdanker.com

**Reply by jdanker** [reply]
Yeah you are right, weird. not seeing where this http:// request for favicon is coming from then.

**Reply by verazuo** [reply]
@jdanker Hi, I meet the same issue and found that you have already solved it. Would you please share your solution? Many thanks!
